site/app/controllers/admin/AdminGradeableController.php

```diff
@@ -83,7 +83,9 @@ public function uploadGradeable(): JsonResponse {
         if (!isset($_POST['id']) || !isset($_POST['title']) || !isset($_POST['type'])) {
             return JsonResponse::getErrorResponse('JSON requires id, title, and type. See documentation for information');
         }
-
+        if (isset($_POST['rubric'])) {
+            $values['rubric'] = $_POST['rubric'];
```

Comment:
Is there an object you can create instead of processing an arbitrary array here?  Lots of magic values to deal with here...